src/pages/en/reference/integrations-reference.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Astro Integration API
+---
+
+## Usage
+
+Astro integrations are always added through the `integrations` property in your  `astro.config.js` file. 
+
+There are three common ways to add an integration to your Astro project:
+1. Import an installed npm package integration.
+2. Import your own integration from another file inside your project.
+3. Write your integration inline, directly in your config file.
+
+```js
+// astro.config.js
+import defineConfig from 'astro/define';
+// 1. Imported from an installed npm package
+import installedIntegration from '@astrojs/vue'
+// 2. Imported from a local file
+import myIntegration from './my-integration.js'
+// 3. Written directly in this file
+const myInlineIntegration = {name: 'namespace:id', hooks: { /* ... */ }};
+
+export default defineConfig({
+  integrations: [
+    installedIntegration(), 
+    myIntegration(),
+    myInlineIntegration
+  ]
+})
+```
+
```

Comment:
Maybe some spaces between the various options in the code sample so that they are separated visually? (Just my preference.) Especially since there are 3 imports/statements *we* are adding, and one of them is an import provided by default in the config file.

Also: in the guide, the statement is `import { defineConfig }` and in this code sample, there are no curly braces. Should they be the same?

Reply:
Thank you! curly braces are required, fixed the code snippet